crown-court-proceeding/build.gradle

```diff
@@ -7,6 +7,7 @@ plugins {
     id "org.sonarqube" version "4.0.0.2929"
     id "info.solidsoft.pitest" version "1.9.11"
     id "org.jsonschema2dataclass" version "6.0.0"
+    id 'io.spring.dependency-management' version '1.1.0'
```

Comment:
Why we need io.spring.dependency-management plugin when we have the spring boot plugin?

Reply:
Used internally by Spring FW cloud, so it is required. Created a separate story to potentially fully migrate to Spring Cloud V3